### PR TITLE
 Implement support for YAML and JSON codecs for the file lookup

### PR DESCRIPTION
**Pull request by danielkza**

### CODE REVIEW

File: stacker/lookups/handlers/file.py [hunk 1]
```diff
@@ -107,18 +110,56 @@ def parameterized_codec(raw, b64):
...
+    result = _parameterize_string(raw)
 
     # Note, since we want a raw JSON object (not a string) output in the
     # template, we wrap the result in GenericHelperFn (not needed if we're
     # using Base64)
-    return Base64(result) if b64 else GenericHelperFn(result)
+    return Base64(result.data) if b64 else result
+
+
+def _parameterize_obj(obj):
```
Comment (phobologic) [inline stacker/lookups/handlers/file.py]: Can we get some docstrings in here?

**Comment by danielkza** [comment]
Rebased and added docstrings to most of the functions. @phobologic

**Comment by phobologic** [comment]
Ahh, issues with a long line from lint.  Tried to fix it, but couldn't push to your branch @danielkza - simple fix, can you clean that up and then I'll merge?  Thanks!

**Comment by danielkza** [comment]
Rebased and hopefully fixed the warning :)

**Comment by phobologic** [comment]
Looks great, thanks @danielkza

**Pull request merged**